Bottom: 353511dbae25c0e293db397b2c7308ada2ecd01e
Top:    7b3ac4d0dce62ff2da7e909649f7195b0adb0436
Author: Peter Clifton <peter@clifton-electronics.co.uk>
Date:   2015-03-21 11:51:00 +0000

lesstif: Use Coord type for line-width, not int


---

diff --git a/src/hid/lesstif/main.c b/src/hid/lesstif/main.c
index ed10b50..79a914a 100644
--- a/src/hid/lesstif/main.c
+++ b/src/hid/lesstif/main.c
@@ -52,7 +52,7 @@ typedef struct lesstif_gc_struct
 
   Pixel color;
   const char *colorname;
-  int width;
+  Coord width;
   EndCapStyle cap;
   char xor_set;
   char erase;
